.env.example

```
@@ -56,6 +56,10 @@ GITHUB_TOKEN="github_pat_"
 # Used for AI chat functionality
 OPENAI_API_KEY=""
 
+# Google Generative AI
+# Used for Moderation
+GOOGLE_GENERATIVE_AI_API_KEY=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)